src/api/ApiTypes.d.tsx

```diff
@@ -10,7 +10,8 @@ export enum RequestType {
     TRACK_SEARCH = "trackSearch",
     PLAYER_NAME = "playerName",
     SET_CONNECTION_ID = "setConId",
-    GET_VERSION = "version"
+    GET_VERSION = "version",
+    FCM_TOKEN = "token"
```

Comment:
remove unused apitype

Reply:
how can it be used from the preScript?

Reply:
it cant, the pre script is outside of typescript/react etc.